packages/experiments/src/components/Shimmer/ShimmerCircle/ShimmerCircle.styles.ts

```diff
@@ -0,0 +1,74 @@
+import * as React from 'react';
+import {
+  IStyle,
+  DefaultPalette,
+  IStyleSet
+} from '../../../Styling';
+import { IStyleFunction } from '../../../Utilities';
+
+export interface IShimmerCircle {
+
+}
+
+/**
+ * ShimmerCircle component props.
+ */
+export interface IShimmerCircleProps extends React.AllHTMLAttributes<HTMLElement> {
+  /**
+   * Optional callback to access the IShimmerCircle interface. Use this instead of ref for accessing
+   * the public methods and properties of the component.
+   */
+  componentRef?: (component: IShimmerCircle) => void;
+
+  /**
+   * Sets the height of the circle.
+   * @default 24px
+   */
+  height?: number;
```

Comment:
Should we name this diameter, as it might make more sense for a circle? And also I notice we are using the value for both width and height in the component.

Reply:
I think it should be left as it is cause the user should think in terms of the height of the entire Shimmer, plus it is used in the calculation of the maxHeight and it would add unnecessary logic.